tests/framework/extensions/provisioning/verify.go

```diff
@@ -20,8 +20,11 @@ import (
 	"github.com/rancher/rancher/tests/framework/extensions/provisioninginput"
 	psadeploy "github.com/rancher/rancher/tests/framework/extensions/psact"
 	"github.com/rancher/rancher/tests/framework/extensions/registries"
+	"github.com/rancher/rancher/tests/framework/extensions/sshkeys"
 	"github.com/rancher/rancher/tests/framework/extensions/workloads/pods"
+	"github.com/rancher/rancher/tests/framework/pkg/nodes"
 	"github.com/rancher/rancher/tests/framework/pkg/wait"
+	"github.com/rancher/rancher/tests/v2/validation/provisioning"
```

Comment:
I missed this. Tim the framework should never import things from v2/validations. I would make sense to move ssh.go to framework/extensions/provisioning.